livingatlas/pipelines/src/main/resources/stateProvinceCentrePoints.txt

```diff
@@ -1,6 +1,7 @@
 Australian Capital Territory	-35.4734679	149.0123679	-35.1245129	149.3992849	-35.9205308	148.7640971
 New South Wales	-31.2532183	146.921099	-28.1561921	153.903718	-37.5052772	140.9992122
 Northern Territory	-19.4914108	132.5509603	-10.790725	137.9990091	-26.0168697	129.0004244
+The Northern Territory	-19.4914108	132.5509603	-10.790725	137.9990091	-26.0168697	129.0004244
 Queensland	-20.9175738	142.7027956	-9.735481199999999	153.7637489	-29.1785876	137.9945749
```

Comment:
Is this right, since ACT could also be The Australian Capital Territory?

Reply:
I think this is to align with the values from the layer where "The Northern Territory" comes back for NT.